frontend/app/package.json

```diff
@@ -55,33 +55,33 @@
     "vue-i18n": "8.27.0",
```

Comment:
Was the new minor version of vue causing issues?

Reply:
I'm pretty sure I tried it, and it was broken.

Reply:
ok

Reply:
It seems that it gives to many errors, which is interesting. I wonder which change causes it. Updating vue-tsc doesn't make a difference either.